### ISSUE TITLE
Unable to build project under WSL2

### ISSUE THREAD

**Issue by davepinto** [ytmdesktop/ytmdesktop#860]
- [x] I understand that **YTMDesktop have NO affiliation with Google or YouTube**.
- [x] I verified that there is no open issue for the same subject.

**Describe the bug**
Unable to build the project under WSL2.

**To Reproduce**
1. Install WSL2
2. Clone repository and install all dependencies
3. run `npm install`

**Expected behavior**
npx electron-builder install-app-deps successfully rebuilds native dependencies, and continues with installation.

**Screenshots**
![unable_to_install](https://user-images.githubusercontent.com/19878677/139528766-4335db3b-148b-4264-8905-6751d85d1b6c.png)

**Environment (please complete the following information):**
 * YTMDesktop version: 1.13.0
 * OS: Debian Linux under WSL2
 * OS version: Debian 11, Windows 11
 * Arch: x64
 * Installation way: From repository
 * Node Version: 16.13.0

Tested with Ubuntu 20.04 under WSL2 as well, same issue.

**Comment by Alipoodle** [comment]
Not 100% sure we are supporting of Node 16 at the moment (I've personally had success with 10 & 12), which maybe where some of the issues are coming from.

And you've not provided Actual Results, but given the screenshot I assume that while installing Dependencies it just freezes and never finishes?

**Comment by davepinto** [comment]
Yes that is the issue, thanks Alipoodle. I was able to install fine on Node V12. Maybe this should be mentioned in the readme, under the section mentioning WSL?

I immediately ran into another issue, due to running under WSL. As it believes it is running under linux, it expects dbus to be there to start mpris. Hacked a fix by just checking if the $WSL_DISTRO_NAME variable is set before starting mpris, but worth noting for anyone else building under WSL.

**Comment by Alipoodle** [comment]
@NovusTheory might be worth looking at the thing above for your Xosms Integration for Linux.

And I'll add the thing for NodeJS Version. (it's not specific to WSL, it's the full thing, although we're currently looking to upgrade it with Electron)

**Comment by Alipoodle** [comment]
Considering you're launching under WSL, NPM has detected it as Linux (which is assumed to be expected).
This seems a bit strange to want to run this under WSL as you're loosing a number of Windows Specific features like Windows Media service.

**Comment by davepinto** [comment]
You're right. I'd like to build and test for windows, but have my entire development toolchain on Linux. Don't know if there's a way to `npm start` and have it build and test a windows version of the app.

**Comment by XeroxDev** [comment]
Yes, it is possible, but you need to have nodejs installed on windows, the files have to be on windows too and you must run the command with the windows console / npm from windows.

Development with wsl is still possible. The windows OS gets mounted on wsl (see /mnt/)

**Comment by Alipoodle** [comment]
Closing issue due to no longer supportinng YouTube Music Desktop v1 + Release of YouTube Music v2.
Please re-open this issue if this is still occuring in the latest build of the application, however as a re-write has been done on the entire project most issues will be fixed, no longer a feature or re-worked.

I have previously tried building YTMDv2 in WSL2 and it had worked.
However it is not recommended to Build it in it still due to the lack of the full feature set of Linux while in Windows.
